source/kmip.rst

```diff
@@ -48,18 +43,51 @@ Starting with release 5.0.8-7, the support for `master key rotation <https://www
       * - --kmipRotateMasterKey
         - boolean
         - Controls master keys rotation. When enabled, generates the new master key version and re-encrypts the keystore. Available as of version 5.0.8-7. Requires the unique ``--kmipKeyIdentifier`` for every ``mongod`` node.
-          
-.. admonition:: Config file example
-
-   .. code-block:: yaml
-
-      security:
-        enableEncryption: true
-        kmip:
-          serverName: 128.0.0.2
-          port: 5696
-          clientCertificateFile: /path/client_certificate.pem
-          clientKeyFile: /path/client_key.pem
-          serverCAFile: /path/ca.pem
-          keyIdentifier: key_name
+
+Key rotation
+================
+
+Starting with release 5.0.8-7, the support for `master key rotation <https://www.mongodb.com/docs/manual/tutorial/rotate-encryption-key/#kmip-master-key-rotation>`_ is added. This enables users to comply with data security regulations when using KMIP.
+
+.. note::
+
+   To make KMIP master key rotation, make sure that every ``mongod`` has a unique ``--kmipKeyIdentifier`` value.
+
+Configuration
+=============
+
+.. rubric:: Considerations
+
+* Make sure you have obtained the root certificate, and the keypair for the KMIP server and the ``mongod`` client. For testing purposes you can use the `OpenSSL <https://www.openssl.org/>`_ to issue self-signed certificates. For production use we recommend you use the valid certificates issued by the key management appliance.
+* The setup of the KMIP server is out of scope of this document. The use of KMIP protocol for data-at-rest encryption is tested with the `PyKMIP server <https://pykmip.readthedocs.io/en/latest/server.html>`__ and the `HashiCorp Vault Enterprise KMIP Secrets Engine <https://www.vaultproject.io/docs/secrets/kmip>`__ so you can use either of these.
```

Comment:
IMHO, I'd remove part that we tested feature only with specific software, any software that provides support for KMIP is expected to be compatible